qa/L0_metrics/test.sh

```diff
@@ -94,6 +94,33 @@ kill $SERVER_PID
 wait $SERVER_PID
 
 
+# Test metrics interval by querying host and checking energy
+METRICS_INTERVAL_SECS=0.5
+SERVER_ARGS="$SERVER_ARGS --metrics-interval-secs=${METRICS_INTERVAL_SECS}"
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+  echo -e "\n***\n*** Failed to start $SERVER\n***"
+  cat $SERVER_LOG
+  exit 1
+fi
+
+num_iterations=10
+prev_energy=`curl -s localhost:8002/metrics | awk '/nv_energy_consumption{/ {print $2}'`
+for (( i = 0; i < $num_iterations; ++i )); do
+  sleep $METRICS_INTERVAL_SECS
+  current_energy=`curl -s localhost:8002/metrics | awk '/nv_energy_consumption{/ {print $2}'`
+  if [ $current_energy == $prev_energy ]; then
```

Comment:
What if the updated value is the same as previous value?

Reply:
The assumption is that the energy consumed will increase over time, monotonically